pilot/pkg/xds/ads_test.go

```diff
@@ -223,8 +225,8 @@ func TestAdsReconnectAfterRestart(t *testing.T) {
 	_ = adscon.CloseSend()
 	adscon = s.ConnectADS()
 
-	// Connect with empty resources.
-	err = sendEDSReq([]string{}, sidecarID(app3Ip, "app3"), res.VersionInfo, res.Nonce, adscon)
+	// Connect with the same resources
```

Comment:
Is n't it the same case as above?

Reply:
Yes, this is about a reconnect. It is intentionally the same. The previous test was wrong, envoy will not reconnect with empty resource.